### PR TITLE
support for delete signatures

### PR DESCRIPTION
**Pull request by lagartoverde**

### CODE REVIEW

**Review by usrtim — APPROVED** [review]
LGTM

**Comment by abeforgit** [comment]
linting fix in #102 if approved, if not will fix manually

File: support/pre-importer.js [hunk 1]
```diff
@@ -1,55 +1,93 @@
...
         SELECT ?content ?physicalFileUri
         WHERE {
          OPTIONAL { ${sparqlEscapeUri(uri)} ${contentPredicate} ?content. }
-         OPTIONAL { ${sparqlEscapeUri(uri)} prov:generated/^nie:dataSource ?physicalFileUri. }
-        }`);
+         OPTIONAL { ${sparqlEscapeUri(
+           uri
+         )} prov:generated/^nie:dataSource ?physicalFileUri. }
+        }`;
+  console.log("contentQuery", contentQuery);
```
Comment (nvdk) [inline support/pre-importer.js:47]: log statement can probably go?

File: models/signed-resource.js [hunk 2]
```diff
@@ -79,5 +143,85 @@ export default class SignedResource {
     this.signatoryRole = signatoryRole;
     this.hashAlgorithm = hashAlgorithm;
     this.hashValue = hashValue;
+    this.deleted = deleted;
+    this.blockchainStatus = blockchainStatus;
```
Comment (nvdk) [inline models/signed-resource.js:147]: blockChainStatus is never loaded/set

File: models/signed-resource.js [hunk 3]
```diff
@@ -1,75 +1,139 @@
...
+             sign:text ?html;
+             sign:hashValue ?hashValue;
+             sign:hashAlgorithm ?hashAlgorithm;
+             dct:created ?created;
+             sign:signatoryRoles ?signatoryRole.
+        
+        ?uri sign:status ?blockchainStatus;
+             sign:signatory ?signatory.
+        
+        ?blockchainStatus mu:uuid ?blockchainStatusUuid.
```
Comment (nvdk) [inline models/signed-resource.js:27]: I'd blockchainStatus and uuid legacy and no longer required

Comment (abeforgit) [reply models/signed-resource.js:27]: in the interest of limiting scope, OK to leave it for now and open a separate pr to clean up obsolete stuff?

**Pull request merged**